### PR TITLE
build/custom: wishlist items

### PR DESCRIPTION
**Pull request by sosiouxme**
Add some features to the custom job that I've really wanted.

Divided into commits for easier reviewing; you can also try it out at https://buildvm.openshift.eng.bos.redhat.com:8443/job/hack/job/lmeyer-aos-cd-builds/job/build%252Fcustom/

### CODE REVIEW

**Comment by sosiouxme** [comment]
@adammhaile @tbielawa

File: jobs/build/custom/Jenkinsfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ node {
                     commonlib.oseVersionParam('BUILD_VERSION'),
                     [
                         name: 'VERSION',
-                        description: 'Version string for build without leading "v" (i.e. 4.0.0)',
+                        description: 'Version string for build (e.g. 4.0.0)',
```
Comment (adammhaile) [inline jobs/build/custom/Jenkinsfile]: I liked that note. Because I did it too many times :P

File: pipeline-scripts/commonlib.groovy [hunk 2]
```diff
@@ -16,12 +16,6 @@ ocpVersions = [
     "3.1",
 ]
 
-ocp4DefaultVersion = "4.0"
```
Comment (adammhaile) [inline pipeline-scripts/commonlib.groovy]: You gotta rebase! You just removed what I just added.

File: jobs/build/custom/Jenkinsfile (hunk 1 above)
Comment (sosiouxme) [reply jobs/build/custom/Jenkinsfile]: The change here is that it works with or without the "v" so you don't have to remember

File: pipeline-scripts/commonlib.groovy (hunk 2 above)
Comment (sosiouxme) [reply pipeline-scripts/commonlib.groovy]: Yeah! sorry 'bout that.

**Comment by sosiouxme** [comment]
Updated for re-review

**Comment by sosiouxme** [comment]
Heh, my bad. The changes aren't too big though. I saw a way to make the regex bit work the way I want, and I added the last commit so build name/description get updated

**Pull request merged**